torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -643,6 +808,14 @@ def center_crop_segmentation_mask(segmentation_mask: torch.Tensor, output_size:
     return center_crop_image_tensor(img=segmentation_mask, output_size=output_size)
 
 
+def center_crop(inpt: DType, output_size: List[int]) -> DType:
+    if isinstance(inpt, features._Feature):
+        return inpt.center_crop(output_size)
+    elif isinstance(inpt, PIL.Image.Image):
```

Comment:
Just a minor nit, but we should decide if we will use the if/if/return idiom or if/elif/else or if/elif/return and keep the same everywhere. 

The changes still LGTM overall.